test/test-cmake.sh

```diff
@@ -41,9 +41,7 @@ EXEC "" ninja -v
 
 # Rerun ninja to make sure that dependencies aren't broken.
 EXEC ninja-rerun ninja -d explain -v
-DIFF ninja-rerun.err - <<EOF
-EOF
-DIFF ninja-rerun.out - <<EOF
+DIFF ninja-rerun.out - <<EOF || cat ninja-rerun.err
```

Comment:
The stderr output is no longer empty with module dependency scanning enabled. It always outputs
```
ninja explain: loading dyndep file 'CMakeFiles/__cmake_cxx23.dir/CXX.dd'
```